client/question/components/MonospaceDisplayModalDirective.js

```diff
@@ -0,0 +1,129 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the modal displaying monospaced code/error
+ * messages.
+ */
+
+tie.directive('monospaceDisplayModal', [function() {
+  return {
+    restrict: 'E',
+    scope: {},
+    template: `
+      <div class="container">
```

Comment:
Shouldn't we use the same "tie-" naming conventions here in this file for the elements? "container", "body" and "content" are generic.

Reply:
Done.